src/components/week/calendarWeekView.component.ts

```diff
@@ -329,6 +336,16 @@ export class CalendarWeekViewComponent implements OnChanges, OnInit, OnDestroy {
    * @hidden
    */
   dragStart(weekViewContainer: HTMLElement, event: HTMLElement): void {
+
+    // With a scrollbar during the drag, the event is only visible inside the calendar.
+    // The "fixed" position bring the event on top, even when dragged outside the calendar.
+    if (this.allowDragOutside && this.maxHeight) {
+      event.style.left = ((event.getBoundingClientRect().left / document.body.clientWidth ) * 100) + '%';
```

Comment:
`getBoundingClientRect()` is quite expensive, can you compute it just once on the line above. Rather than using a percentage can it not just be `event.getBoundingClientRect().left + 'px'` ? What happens if the user cancels and the event isn't dropped out of the calendar? 

Maybe it's better to put this functionality into the drag and drop module as this would be helpful to anyone using drag and drop within a scrollable box. What do you think?